README.rst

```diff
@@ -0,0 +1,39 @@
+=========
```

Comment:
For python projects `.rst` is convenient as we can parse the file and use the content in the `setup.py` for PyPI